### ISSUE TITLE
Explicações para tipos de canais

### ISSUE THREAD

**Issue by cv** [servicosgovbr/editor-de-servicos#248]
Nos tipos de canais não há uma explicação. Por exemplo, no tipo do canal web não informa que no campo abaixo tem que ser informada uma url.

**Comment by cv** [comment]
Mudar apenas o label ("Informe o link", "Informe o telefone no formato xxx", etc. de acordo com o tipo selecionado.

**Comment by sahrosa** [comment]
A ideia é que tenhamos um campo markdown aberto mesmo, para que possa ser preenchido a informação mais precisa possível, dentro das suas possibilidades. Ex: canal presencial pode ser escrito o endereço do local ou um link para um localizador de agências, etc.

Acham que é prioridade 0 modificar os labels?

**Comment by cv** [comment]
Sim, @sahrosa - mas apenas os labels. O resto fica igual (markdown, campos abertos, etc)

**Comment by sahrosa** [comment]
'Aplicativo móvel' - Escreva o nome e link para o download do aplicativo
    'E-mail' - Escreva o endereço de e-mail que deverá ser utilizado
    'Fax' - Escreva o número para qual o fax deverá ser enviado
    'Postal' - Escreva o endereço de e-mail que deverá ser utilizado
    'Presencial' - Escreva o endereço que deverá ser visitado 
    'SMS' - Escreva o número para qual o SMS deverá ser enviado
    'Telefone' - Escreva o número para qual a ligação deverá realizada 
    'Web' - Escreva a url do site que deverá ser visitado
    'Web: Acompanhar' - Escreva o link que deverá ser visitado para acompanhar esta etapa
    'Web: Agendar' - Escreva o link que deverá ser visitado para agendar esta etapa
    'Web: Calcular taxas' - Escreva o link que deverá ser visitado para calcular taxas nesta etapa
    'Web: Consultar' - Escreva o link que deverá ser visitado para consultar nesta etapa
    'Web: Declarar' - Escreva o link que deverá ser visitado para declarar nesta etapa
    'Web: Emitir' - Escreva o link que deverá ser visitado para emitir nesta etapa
    'Web: Inscrever-se' - Escreva o link que deverá ser visitado para inscrever-se nesta etapa
    'Web: Preencher' - Escreva o link que deverá ser visitado para preencher nesta etapa
    'Web: Simular' - Escreva o link que deverá ser visitado para simular esta etapa

**Comment by sahrosa** [comment]
@cv @kchnr vocês me ajudam depois a conectar os textos aos selects? Hoje temos apenas um texto "descreva como usar este canal" para todos os casos.

**Comment by cv** [comment]
Deixa que eu chuto ;)

(algo pra se fazer no avião :P)

2015-08-21 18:09 GMT-03:00 Samantha Rosa <notifications@github.com>:
